src/store/analyticsSlice.ts

```diff
@@ -42,10 +42,14 @@ export const createAnalyticsSlice: StateCreator<
       const eventProperties = {
         ...properties,
         walletAddress: get().account,
+        market: get().currentMarket,
+        walletType: get().walletType,
       };
 
       try {
-        mixpanel.track(eventName, eventProperties);
+        const result = get();
+        if (result && result.currentMarket && result.currentMarket != 'proto_scroll_alpha_v3')
```

Comment:
if currentMarket is undefined basically no mixpanel events will fire.

Reply:
@0x4Graham for the `proto_scroll_alpha_v3` can you add that as an array like `const ELXLUDED_MARKETS = ['proto_scroll_alpha_v3']` and then you can use `includes()` so we can add additional

Reply:
Fixed